sigh/lib/assets/resign.sh

```diff
@@ -147,15 +183,17 @@ for provision in ${RAW_PROVISIONS[@]}; do
         log "Specified provisioning profile: '$provision'"
     fi
 done
+
+log "Original file: ${ORIGINAL_FILE}"
+log "Certificate: ${CERTIFICATE}"
 [[ -n "${DISPLAY_NAME}" ]] && log "Specified display name: '$DISPLAY_NAME'"
 [[ -n "${ENTITLEMENTS}" ]] && log "Specified signing entitlements: '$ENTITLEMENTS'"
 [[ -n "${BUNDLE_IDENTIFIER}" ]] && log "Specified bundle identifier: '$BUNDLE_IDENTIFIER'"
-[[ -n "${KEYCHAIN}" ]] && log "Specified Keychain to use: '$KEYCHAIN'"
-[[ -n "${VERSION_NUMBER}" ]] && log "Specified version to use: '$VERSION_NUMBER'"
-
-shift $((OPTIND-1))
+[[ -n "${KEYCHAIN}" ]] && log "Specified keychain to use: '$KEYCHAIN'"
+[[ -n "${VERSION_NUMBER}" ]] && log "Specified short version to use: '$VERSION_NUMBER'"
+[[ -n "${NEW_FILE}" ]] && log "Output file name: '${NEW_FILE}'"
```

Comment:
Should this be 
`[[ -n "${NEW_FILE}" ]] && log "Output file name: '$NEW_FILE'"` 
instead of 
`[[ -n "${NEW_FILE}" ]] && log "Output file name: '${NEW_FILE}'"`